models/revoked_tokens.py

```diff
@@ -5,7 +5,7 @@
 class RevokedTokensModel(BaseModel):
     __tablename__ = "revoked_tokens"
 
-    id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.String(36), primary_key=True)
```

Comment:
I don't think the solution is to change the id field to a String. That does make the test pass, but will cause other issues that we are apparently missing tests for.